command/assets/Format.go

```diff
@@ -0,0 +1,22 @@
+package assets
+
+import (
+	"math"
+	"strconv"
+	"unicode/utf8"
+)
+
+func PaddingRight(val string, length int, text string) string {
+	for i := utf8.RuneCountInString(val); i < length; i++ {
+		val += text
+	}
+	return val
```

Comment:
`text` が二文字以上の時がカバーされてない気がする
(`strings` 使いました)
```suggestion
  remaining := length - utf8.RuneCountInString(val)
  if remaining <= 0 {
    return val
  }
	return val + strings.Repeat(text, remaining / utf8.RuneCountInString(text))
```